modules/mod.rb

```diff
@@ -74,6 +74,40 @@ module Mod
 				event.respond('❌ Invalid argument. Please mention a valid user.')
 			end
 		end
+	  
+		command(:warn, description: 'Warn somebody on the server. Mod only.', usage: "#{Config.prefix}warn @user reason", min_args: 2) do |event, *kick_reason|
+			unless Helper.is_helper?(event) || Helper.is_moderator?(event) || Helper.is_developer?(event) || Helper.is_admin?(event.user)
+				event.respond("âŒ You don't have permission for that!")
```

Comment:
GitHub is having issues displaying them, but they show correctly outside of it, ignore